### ISSUE TITLE
Location of setting for global post privacy is not documented. Happy to contribute! 

### ISSUE THREAD

**Issue by PrecisionWordcraft** [mastodon/documentation#1041]
Hi,

I think I spent about an hour looking in the docs, and then in the UI, for the setting that controls global post privacy. I finally found it but it was painful.

I'm happy to submit a pull request for a doc change that tells users exactly where the setting is. May I simply submit a pull request, or should I fork the repo and do it from there?

Thanks!

**Comment by trwnh** [comment]
Do you mean for the user guide?

**Comment by PrecisionWordcraft** [comment]
Yes - [this section of this page](https://docs.joinmastodon.org/user/preferences/#misc). 

I'm about to submit a pull request on a new branch.
